client/package.json

```diff
@@ -16,11 +17,12 @@
     "react-scripts": "3.4.1",
     "redux": "^4.0.5",
     "redux-saga": "^1.1.3",
+    "tailwindcss": "^1.4.6",
     "thunk": "0.0.1",
     "uuidv4": "^6.0.9"
   },
   "scripts": {
-    "build": "react-scripts build",
+    "build": "npx tailwindcss build src/tailwind.css -o src/styles.css && react-scripts build",
```

Comment:
🔮 elsewhere we use `npm` -- why use `npx` here?

Reply:
https://stackoverflow.com/questions/50605219/difference-between-npx-and-npm